grafana-plugin/package.json

```diff
@@ -135,7 +136,7 @@
     "@grafana/data": "^11.1.3",
     "@grafana/faro-web-sdk": "^1.4.2",
     "@grafana/faro-web-tracing": "^1.4.2",
-    "@grafana/labels": "~1.5.1",
+    "@grafana/labels": "~1.6.0",
```

Comment:
I guess this closes out https://github.com/grafana/oncall/pull/4900? (cc: @teodosii)

Reply:
yes, thanks!

Reply:
I didn't merge it because I couldn't test it locally. Was it tested?